Readme.md

```diff
@@ -165,8 +165,54 @@ It is not a problem to render async components with `react-snap`, the tricky par
 
 Usually _code splitting_ library provides an API to handle it during SSR, but as long as "real" SSR is not used in react-snap - the issue surfaces, and there is no simple way to fix it.
 
-1. use `loadable-components` 2.2.3 (current is >5)
-The old version of `loadable-components` can solve this issue for a "snapshot" setup:
+1. Use [react-prerendered-component](https://github.com/theKashey/react-prerendered-component)
+   This library could _keep_ the pre-rendered HTML until codesplitted part is ready.
+
+```js
+import loadable from "@loadable/component";
+import { PrerenderedComponent } from "react-prerendered-component";
+
+const prerenderedLoadable = dynamicImport => {
+  const LoadableComponent = loadable(dynamicImport);
+  // you can use `.preload()` method for react-loadable or react-imported-component
+  const live = () => LoadableComponent.load();
+  return React.memo(props => (
+    <PrerenderedComponent live={live}>
+      <LoadableComponent {...props} />
+    </PrerenderedComponent>
+  ));
+};
+
+const MyComponent = prerenderedLoadable(() => import("./MyComponent"));
+```
+
+`MyComponent` will use prendered html to prevent flash (it will find needed piece of html using `id` generated by `PrerenderedComponent` and inject it using `dangerouslySetInnerHTML`).
```

Comment:
`MyComponent` will use prerendered HTML to prevent the page content from flashing (it will find the required piece of HTML using an `id` attribute generated by `PrerenderedComponent` and inject it using `dangerouslySetInnerHTML`).